### PR TITLE
Intermediate HTML and CSS Course - Custom Properties lesson: fix inaccurate statement about codepen configuration

### LINKED ISSUE #27111: Custom Properties: <Correct an inaccurate statement>
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

"First we used JavaScript to add a class attribute on our html element (We don’t have direct access to the html element in the HTML tab on codepen, but you should be able to see the class being applied in the browser’s inspector) so that our page has a default theme."

This is not accurate, the JS changes the class but does not apply a default one.  A setting (using the cogwheel button on codepen) in the HTML tab has explicitly stated that the html should have a class of "dark". This is evident upon opening this code in VS Code, it will not have a default class and what appears to be the light theme will have no styling. If the JS actually applied a default class it would do so in an IDE also. 

Perhaps reword this to : 
"First we applied a default "dark" class to our html element using the settings button on the HTML tab (We don't have direct access to the html element in the HTML tab on codepen, but you should be able to see the class being applied in the browser’s inspector) so that our page has a default theme."

This change would simply be for accuracy, and not having the reader struggling to find how the JavaScript is supposedly applying the default class.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-custom-properties

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

This is my first time I hope I didn't mess up anything (per the Contributing Guide). Just trying to be helpful.

### PR DESCRIPTION
**Pull request by Jaytpa01**
## Because

As per the issue: This change would simply be for accuracy, and not having the reader struggling to find how the JavaScript is supposedly applying the default class.

## This PR

- Changes the statement about how the default class is applied to the root element of the codepen
- Encourages the user to experiment with this setting and how it effects the theme / styles

## Issue

Closes #27111 

## Additional Information

In a seperate pr #27130 I ran into some markdown linting issues, so I've also fixed up any issues I've noticed with this file

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by Jaytpa01** [comment]
There is still a markdown linting issue relating to content `Required heading structure [Expected: ### Lesson overview; Actual: ### Learning outcomes]`

Would rewording `Learning outcomes` to `Lesson overview` suffice, or should a new section be added? It might even make sense to merge these two sections

If you'd like to merge/add a new one, I've prepared a suggestion:

```
### Lesson overview

This section contains a general overview of topics that you will learn in this lesson.

- How to declare and use custom properties
- The scope of custom properties
- The root selector
- Using media queries
```

**Comment by wise-king-sullyman** [comment]
"Lesson overview" should replace "Learning outcomes", your suggestion sounds good to me, do you want to do it as part of this PR or under a new one?

**Comment by Jaytpa01** [comment]
Let's do it under this pr, it'll probably be simpler that way

I'll update the pr in a few hours when I'm at my computer!

**Comment by Jaytpa01** [comment]
Okay @wise-king-sullyman those changes have been pushed!

Let me know if there's anything else that needs to be done before merging this

**Review by wise-king-sullyman — APPROVED** [review]
🔥

**Comment by wise-king-sullyman** [comment]
Looks great, thanks for your work on this!
